test/itkVnlForward1DFFTImageFilterTest.cxx

```diff
@@ -43,22 +43,24 @@ int itkVnlForward1DFFTImageFilterTest( int argc, char* argv[] )
   typedef itk::Image< std::complex< PixelType >, Dimension >                            ComplexImageType;
 
   typedef itk::ImageFileReader< ImageType >                                             ReaderType;
-  typedef itk::VnlForward1DFFTImageFilter< ImageType, ComplexImageType > FFTType;
-  typedef itk::ComplexToRealImageFilter< ComplexImageType, ImageType >                  RealFilterType;
-  typedef itk::ComplexToImaginaryImageFilter< ComplexImageType, ImageType >             ImaginaryFilterType;
-  typedef itk::ImageFileWriter< ImageType >                                             WriterType;
-
   ReaderType::Pointer reader = ReaderType::New();
-  FFTType::Pointer    fft    = FFTType::New();
-  RealFilterType::Pointer realFilter = RealFilterType::New();
-  ImaginaryFilterType::Pointer imaginaryFilter = ImaginaryFilterType::New();
-  WriterType::Pointer writer = WriterType::New();
-
   reader->SetFileName( argv[1] );
+
+  typedef itk::VnlForward1DFFTImageFilter< ImageType, ComplexImageType > FFTType;
+  FFTType::Pointer    fft    = FFTType::New();
   fft->SetInput( reader->GetOutput() );
+
+  typedef itk::ComplexToRealImageFilter< ComplexImageType, ImageType >                  RealFilterType;
```

Comment:
Is this space intentional?

Reply:
Is was intentional when originally added.

There are more style updates that could be made, but I will leave that to another day. 🌤 

Thanks for the review! 👍